packages/lesswrong/lib/abTests.ts

```diff
@@ -33,3 +33,19 @@ export const collectionsPageABTest = new ABTest({
     },
   }
 });
+
+// A/B test for the new BooksProgressBar
+export const booksProgressBarABTest = new ABTest({
+  name: "booksProgressBarABTest",
+  description: "Tests the new BooksProgressBar, as used in BooksItem (itself used in CollectionsPage)",
+  groups: {
+    control: {
+      description: "Original BooksItem without the progress bar",
+      weight: 1
+    },
+    progressBar: {
+      description: "Progress bar enabled",
+      weight: 1
+    }
+  }
+})
```

Comment:
[Nitpick] [Files should end with newlines](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline). That one of the few autoformats we haven't turned off here on ForumMagnum, and so will cause spurious diff if left in.

Reply:
Done, thanks